src/platform_impl/windows/monitor.rs

```diff
@@ -185,6 +186,32 @@ impl MonitorHandleProvider for MonitorHandle {
             .ok()
     }
 
+    fn physical_size(&self) -> Option<PhysicalSize<u32>> {
+        let monitor_info_ex_w = get_monitor_info(self.0).ok()?;
+
+        let hdc = unsafe {
+            CreateDCW(
+                monitor_info_ex_w.szDevice.as_ptr(),
+                monitor_info_ex_w.szDevice.as_ptr(),
+                std::ptr::null(),
+                std::ptr::null(),
+            )
+        };
+
+        let width: u32 =
+            unsafe { GetDeviceCaps(hdc, HORZSIZE.try_into().ok()?) }.try_into().ok()?;
+        let height: u32 =
+            unsafe { GetDeviceCaps(hdc, VERTSIZE.try_into().ok()?) }.try_into().ok()?;
+
+        unsafe {
+            if DeleteDC(hdc) == 0 {
+                tracing::error!("DeleteDC {:?} failed", hdc)
+            }
+        }
```

Comment:
So, I know next to nothing about the Windows API here, but are you sure this is correct?

Looking around a bit I found https://stackoverflow.com/q/577736, it seems to me that perhaps `CreateDCW` + `GetDeviceCaps` isn't the right way of doing this? Have you tested this in multi-monitor setups? And how does it interact [with different resolution settings / rotated screens](https://support.microsoft.com/en-us/windows/change-your-screen-resolution-and-layout-in-windows-5effefe3-2eac-e306-0b5d-2073b765876b)?

Reply:
Indeed, that was not a right way to do it. It gave wrong value on different resolutions. I rewrote code from that stackoverflow answer and it works correctly.